docs/neo-n3/dapp_development/interacting_with_a_node.md

```diff
@@ -165,23 +137,22 @@ if (response.hasError()) {
 String balance = response.getWalletBalance().getBalance();
 ```
 
-And, in the end, close the wallet.
+Finally, close the wallet:
 
 ```java
 NeoCloseWallet response = neow3j.closeWallet().send();
 if (response.hasError()) {
-    throw new Exception("Failed to close the  wallet. Error message: " + response.getError().getMessage());
+    throw new Exception("Failed to close the wallet. Error message: " + response.getError().getMessage());
 }
 ```
 
 ## Neo-Express
 
-The class `io.neow3j.protocol.Neow3jExpress` extends the methods of `Neow3j` with methods that are specific to
-neo-express. [Neo-express](https://github.com/neo-project/neo-express) is a developer tool that enables a fast workflow. It is basically a tool for managing and configuring private networks for development purposes.
-Neo-express exposes several RPC methods additional to normal Neo nodes. These are available through `Neow3jExpress`. Use `Neow3jExpress` just like `Neow3j` but with an URL that points to a neo-express instance.
+The class `io.neow3j.protocol.Neow3jExpress` extends the methods of `Neow3j` with methods specific to neo-express. [Neo-express](https://github.com/neo-project/neo-express) is a developer tool that facilitates a streamlined workflow, providing tools for managing and configuring private networks for development purposes.
+Neo-express exposes additional RPC methods compared to normal Neo nodes, accessible through `Neow3jExpress`. Use `Neow3jExpress` similarly to `Neow3j`, but with a URL pointing to a neo-express instance.
```

Comment:
THis line intends to be a new paragraph or a continuation of the previous paragraph? If a new paragraph, add a line in-between. If not, then maybe it'd be good to add this to the previous line, after the period.